<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BerekenBTW</class>
 <widget class="QDialog" name="BerekenBTW">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BTW Calculator</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>155</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>146</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bedrag inclusief BTW:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>146</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bedrag exclusief BTW:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbIncl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>15</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbExcl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtn6">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>65</y>
     <width>104</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BTW 6%</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtn21">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>65</y>
     <width>104</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BTW 21%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BerekenBTW</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BerekenBTW</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
